modules/data.atmosphere/R/download.NOAA_GEFS.R

```diff
@@ -43,6 +43,39 @@
 ##' @param ... Other arguments, currently ignored
 ##' @export
 ##' 
+##' Example testcases for running.
+##' \dontrun {
+##'  #This program is to test the R script donload.NOAA.R during development.
+##'
+##' test_no = 1
+##' args = commandArgs(trailingOnly = TRUE)
+##' if (length(args) > 0) {
+##'  test_no = as.integer(args[1])
+##'}
+##'
+##' source("~/pecan/modules/data.atmosphere/R/download.NOAA_GEFS.R")
+##'
+##'
+##' if (test_no == 1) { #Default case
+##'  download.NOAA_GEFS("~/Working/results", lat.in= 46.2420, lon.in = -89.3476, "willow creek")
```

Comment:
1) There needs to be an `@example` tag before the example code
2) I I think this test case is too complicated -- you really just need this one line example within the `\dontrun` block

Reply:
It's actually `@examples` not example